doc/045_working_with_repos.rst

```diff
@@ -252,3 +257,13 @@ check all repository pack files over 5 separate invocations:
     $ restic -r /srv/restic-repo check --read-data-subset=3/5
     $ restic -r /srv/restic-repo check --read-data-subset=4/5
     $ restic -r /srv/restic-repo check --read-data-subset=5/5
+
+Use ``--read-data-subset=n%`` to check a randomly choosen subset of the
+repository pack files. It takes one parameter, ``n``, the percentage of pack
+files to check as a floating point number. This will not guarantee to cover all
+available pack files after sufficient runs, but it is easy to automate checking
+a small subset of data after each backup:
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo check --read-data-subset=2.5%
```

Comment:
Should probably add an example of integer-only percentage as well, just for clarity.

Reply:
Fixed in https://github.com/restic/restic/pull/3038/commits/4943b1ebda8ad41c5cd485e202336e780a1e8c77